### DISCUSSION TITLE
Security measures documentation [Jun 22]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5991] [Discord Threads]
> 🙋‍♂️ **Zheng**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Any specific aspects that you are looking at? If you have a template or a questionnaire - we can check and populate whatever information we are aware of.  
#  
> 🙋‍♂️ **Zheng**  
#### Hi <@857998597474484295> , thank you very much for your reply.   
####    
#### Our use case is to ingest some user survey data that is planned to store in a database running on the same virtual machine with NocoDB.    
####    
#### And for the security review, the security team would like to understand what security measures are in place in the software that safeguards our data. Questions might be like:   
####    
#### **Code Auditing:** Determine if the project has undergone code audits by security experts or if it has a process in place for regular security reviews. Open-source projects with a transparent and well-documented codebase are generally more reliable.   
####    
#### **Vulnerability Management: **Check how the project handles security vulnerabilities. Do they have a clear process for reporting and addressing vulnerabilities? Look for evidence of timely security patches and updates.   
####    
#### **Encryption and Authentication:** Verify if the tool supports strong encryption algorithms and secure authentication mechanisms. Look for features such as Transport Layer Security (TLS) for secure communication.   
####    
#### **Access Control and Authorization:** Ensure the tool provides robust access control mechanisms, allowing administrators to define user roles, permissions, and restrictions appropriately. It should support authentication and authorization protocols like OAuth or LDAP.  
#  
> 🙋‍♂️ **Zheng**  
#### Hi <@857998597474484295>,  would you be able to have some related info for this? Thanks!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1105549215272280094)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
